pagerduty/resource_pagerduty_event_orchestration_path_unrouted_test.go

```diff
@@ -0,0 +1,477 @@
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "type", "unrouted"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.#", "0"),
+				),
+			},
+			{
+				Config: testAccCheckPagerDutyEventOrchestrationPathUnroutedConfigWithConditions(team, escalation_policy, service, orchestration),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckPagerDutyEventOrchestrationPathUnroutedExists("pagerduty_event_orchestration_unrouted.unrouted"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.0.expression", "event.summary matches part 'rds'"),
+				),
+			},
+			{
+				Config: testAccCheckPagerDutyEventOrchestrationPathUnroutedConfigWithMultipleRules(team, escalation_policy, service, orchestration),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckPagerDutyEventOrchestrationPathUnroutedExists("pagerduty_event_orchestration_unrouted.unrouted"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.#", "2"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.0.expression", "event.summary matches part 'rds'"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.1.expression", "event.severity matches part 'warning'"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.1.conditions.0.expression", "event.severity matches part 'info'"),
+				),
+			},
+			{
+				Config: testAccCheckPagerDutyEventOrchestrationPathUnroutedWithAllConfig(team, escalation_policy, service, orchestration),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckPagerDutyEventOrchestrationPathUnroutedExists("pagerduty_event_orchestration_unrouted.unrouted"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.#", "2"),
+					//Set #1
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.id", "start"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.#", "2"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.route_to", "child-1"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.0.expression", "event.severity matches part 'info'"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.conditions.1.expression", "event.severity matches part 'warning'"),
+					resource.TestCheckResourceAttr(
```

Comment:
I took a slightly different path and extracted checking attributes for rule actions in a [separate function](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/ORCA-3465-orchestration-service-resource/pagerduty/resource_pagerduty_event_orchestration_path_service_test.go#L60-L79). But now that I see this approach it seems more readable and maybe we should keep all checks within the test step. I think I'm going to make changes on my branch.